test/test_quantized_models.py

```diff
@@ -0,0 +1,49 @@
+import torch
+import unittest
+import torchvision
+
+
+class Tester(unittest.TestCase):
+    def test_model_quantizable(self):
```

Comment:
While this is a fair approach for testing models, if the test fails it will be fairly difficult to understand which model failed. Plus, it is impossible to only run a particular test.

I see two approaches: do something similar to what we do in `test_models.py`, where we dynamically create new tests for the models, or an approach similar to https://github.com/pytorch/vision/pull/1468, which is more explicit.
Let me know which one you prefer.
This can be done in a separate PR as well if it makes things easier.